### DISCUSSION TITLE
UMEP Tree Generator

### DISCUSSION THREAD

**Discussion by Axou973** [UMEP-dev/UMEP#432] [Q&A]
Hello, I'm trying to creat a CDSM with the tool Tree Generator From UMEP pre-processor but it doesn't work. The generator created two tif : Cdsm and Tdsm but I can't see the trees on the raster. It is just a black raster with no value...
![befor the simulation](https://user-images.githubusercontent.com/91162605/176891142-e595fa20-c461-4ded-a1ce-c17daea730df.jpg)
![explorator](https://user-images.githubusercontent.com/91162605/176891144-dd83898e-70a7-4f8d-895a-a46d9002d556.jpg)
![layer](https://user-images.githubusercontent.com/91162605/176891145-1a8662b7-19e0-49bc-8d11-2de36ab51689.jpg)
![result](https://user-images.githubusercontent.com/91162605/176891146-3bed49ef-0cb8-447f-b8a6-8ebb143ce882.jpg)
![tree generator](https://user-images.githubusercontent.com/91162605/176891147-51384f33-c6b0-4e96-a4b4-8c64bbf0ad8b.jpg)

Do you have any suggestions ?
Thank you

**Comment by biglimp** [comment]
First try the part in our tutorial section and see if you can successfully produce a CDSM and TDSM using our test datasets (https://umep-docs.readthedocs.io/projects/tutorial/en/latest/Tutorials/IntroductionToSolweig.html#the-tree-generator). If that works, you should check the CRS of your data (should preferably be the same for all layers)and also check the units of your CRS used (should be in meter).

**Comment by Axou973** [comment]
Thank you but I don't know how to creat a TDSM... 

Best regards 


De: "biglimp" ***@***.***> 
À: "UMEP-dev/UMEP" ***@***.***> 
Cc: "Axou973" ***@***.***>, "Author" ***@***.***> 
Envoyé: Vendredi 1 Juillet 2022 14:13:21 
Objet: Re: [UMEP-dev/UMEP] UMEP Tree Generator (Discussion #432) 





First try the part in our tutorial section and see if you can successfully produce a CDSM and TDSM using our test datasets ( [ https://umep-docs.readthedocs.io/projects/tutorial/en/latest/Tutorials/IntroductionToSolweig.html#the-tree-generator | https://umep-docs.readthedocs.io/projects/tutorial/en/latest/Tutorials/IntroductionToSolweig.html#the-tree-generator ] ). If that works, you should check the CRS of your data (should preferably be the same for all layers)and also check the units of your CRS used (should be in meter). 

— 
Reply to this email directly, [ https://github.com/UMEP-dev/UMEP/discussions/432#discussioncomment-3063706 | view it on GitHub ] , or [ https://github.com/notifications/unsubscribe-auth/AVXQP3M3LFDP3DQIMMLJY53VR3OGDANCNFSM52MRG5FA | unsubscribe ] . 
You are receiving this because you authored the thread. Message ID: <UMEP-dev/UMEP/repo-discussions/432/comments/3063706 @ github . com> 

-- 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
MONTLOUIS-CALIXTE Andréa 
FI4 groupe TD4 
Etudiante en Energie Bâtiment Environnement 
Polytech Chambéry

**Reply by biglimp** [reply]
A TDSM is automatically created when using the tool. It represents the areas underneath the tree crowns.
